server/test/framework/Regression_Tests.cpp

```diff
@@ -340,6 +340,42 @@ namespace {
                 "f4");
     }
 
+    TEST_F(Regression_Test, Pointers_Alignment) {
+        static const size_t HEX = 16;
+        static const size_t NODE_ALIGNMENT = 8;
+        
+        fs::path source = getTestFilePath("issue-195.c");
+        auto [testGen, status] = createTestForFunction(source, NODE_ALIGNMENT);
+
+        ASSERT_TRUE(status.ok()) << status.error_message();
+
+        std::function<bool(const tests::Tests::TestCaseParamValue &)> checkAlignmentInNode =
+            [&checkAlignmentInNode](const tests::Tests::TestCaseParamValue &param) {
+                const auto stringPointer = param.view->getSubViews().at(1)->getEntryValue(nullptr);
+                const size_t address = strtoull(stringPointer.c_str(), nullptr, HEX);
+                bool result = (address % NODE_ALIGNMENT == 0);
+
+                for (const auto &innerLazyValueParam : param.lazyValues) {
+                    result &= checkAlignmentInNode(innerLazyValueParam);
+                }
+                return result;
+            };
+
+
+        checkTestCasePredicates(
```

Comment:
This check at least one test without misalignment. You need to check that all tests has correct alignment.